### DISCUSSION TITLE
how fix to erorr to Wasm

### DISCUSSION THREAD

**Discussion by AbdalaMask** [unoplatform/uno#8853] [Q&A]
### Current behavior

![image](https://user-images.githubusercontent.com/37013638/169711915-c89ead13-ed55-4109-b400-4502c68e969f.png)

### Anything else we need to know?

![image](https://user-images.githubusercontent.com/37013638/169712010-d7df1435-6094-492a-bef3-8497748dd471.png)
I'm tired of this mistake

**Comment by AbdalaMask** [comment]
![image](https://user-images.githubusercontent.com/37013638/169712611-45c03e2e-7ad0-4d0d-b7fc-2565bb24c229.png)

**Reply by jeromelaban** [reply]
This generally means that emscripten did not install properly. You'll likely need to:

- Delete this folder `%USERPROFILE%\.uno\emsdk`
- Install Python (See https://www.microsoft.com/store/productId/9P7QFQMJRFP7 and https://github.com/unoplatform/Uno.Wasm.Bootstrap/blob/main/doc/runtime-execution-modes.md#runtime-execution-modes)
- Try building again.

**Reply by AbdalaMask** [reply]
![image](https://user-images.githubusercontent.com/37013638/169717643-08df8d9f-7813-419e-b2cb-7bbbf2032262.png)

**Reply by AbdalaMask** [reply]
![image](https://user-images.githubusercontent.com/37013638/169717687-b13946bd-7a40-4cb2-9ee0-a00546dc6cfb.png)

**Reply by AbdalaMask** [reply]
![image](https://user-images.githubusercontent.com/37013638/169717943-8bd8b036-cb41-4a70-8060-d098fdd9ccaf.png)

**Reply by AbdalaMask** [reply]
![image](https://user-images.githubusercontent.com/37013638/169717969-d80ab53e-6aa7-4823-a726-d673eb3becef.png)

**Reply by AbdalaMask** [reply]
![image](https://user-images.githubusercontent.com/37013638/169718263-c0d341e9-522d-4301-aa1d-af6d3291f309.png)
![image](https://user-images.githubusercontent.com/37013638/169718294-0d034050-de2d-4dfa-9ba0-a55544750613.png)
![image](https://user-images.githubusercontent.com/37013638/169718312-67ae8517-3381-4c55-8022-f775a831f7ab.png)

**Reply by jeromelaban** [reply]
Thanks. Please avoid posting images, post actual text or create binlogs using this document: https://platform.uno/docs/articles/uno-builds-troubleshooting.html#generating-msbuild-binary-log-files. If the error happens again, post the binlog.

**Reply by AbdalaMask** [reply]
thank you so much

**Comment by AbdalaMask** [comment]
thank you so much
